packages/create-bison-app/template/tests/e2e/auth.play.ts

```diff
@@ -1,31 +1,39 @@
 // https://playwright.dev/docs/next/auth#multiple-signed-in-roles
 import { Page, test, expect } from '@playwright/test';
+import { Role } from '@prisma/client';
 
-import { ADMIN, APP_URL, USER } from './constants';
+import { loginAs } from './helpers';
+
+import { resetDB } from '@/tests/helpers/db';
+import { disconnect } from '@/lib/prisma';
+
+test.beforeEach(async () => resetDB());
+test.afterAll(async () => disconnect());
 
 test.describe(() => {
-  test.use({ storageState: USER.storageState });
+  test('Can login as a User', async ({ page }: { page: Page }) => {
+    const firstName = 'Qwerty';
+    await loginAs(page, { roles: [Role.USER], firstName });
```

Comment:
At some point, this test should be updated to log the user in using the UI, instead of `loginAs`. But we should have at least one test that uses `loginAs` at that point, so we have an example.